wiki/Internet_Relay_Chat/en.md

```diff
@@ -1,100 +1,84 @@
 What is Internet Relay Chat?
 ============================
 
-The [Internet Relay Chat](http://en.wikipedia.org/wiki/Internet_Relay_Chat), or also known as IRC is a well established standardized protocol for chatting with numerous clients available to connect with.
+The [Internet Relay Chat](http://en.wikipedia.org/wiki/Internet_Relay_Chat), also known as IRC, is a well established standardized protocol for chatting with numerous clients available to connect with.
 
-<center>
-</center>
-osu! Chat
-=========
 
-osu! uses the IRC protocol for its [in-game chat](Chat_Console "wikilink"). You can connect with your own client and chat with your osu! buddies even when you're not running osu!. Note that osu! Bancho uses a custom implementation of the IRC protocol, and is in no way a full-feature implementation; therefore, do not expect all features of your IRC client to function properly.
+osu!Bancho
+---------
+osu!Bancho uses the IRC protocol for its in-game chat. You can connect with your own client and chat with other people without using the game client. Take note that this IRC protocol is modified therefore do not expect your client's features to work properly.
 
 **Notice: [HexChat](http://hexchat.github.io/) is known to have problems with osu!'s IRC** ([bug report of HexChat's GitHub](http://github.com/hexchat/hexchat/issues/818)), consider using another client if it bothers you.
 
+
 How To Connect
 --------------
+Once you have a client, you need to set up the server settings.
 
-Once you have a client you have to use your osu! username as nickname and you can connect to
-
-` `[`cho.ppy.sh`](irc://cho.ppy.sh)` OR `[`irc.ppy.sh`](irc://irc.ppy.sh)` (both resolve to the same host) on port `<b>`6667`</b>` (the default IRC port)`
-
-Authenticating To Bancho
-------------------------
-
-When you first connect you'll get a message like this.
+- **Server:** `cho.ppy.sh` or `irc.ppy.sh`
+- **Port:** `6667` (default)
+- **Username:** Your osu! username (replace spaces to underscores)
+- **Password:** Get it at [IRC Authentication](https://osu.ppy.sh/p/irc).
 
-`* Welcome to osu!bancho.`
-`* -`
-`* - You are required to authenticate before accessing this service.`
-`* - Please click the following link to complete this process:`
+*Your IRC password is different from your account password. **Do not share it to others**.*
 
-When you go to the URL you are presented with you'll see a screen with a big fancy "Authorise IRC connection" button. Simply clicking this will allow you into the chat and auto-join you into [\#osu](irc://cho.ppy.sh/osu).
-
-If you wish to not have to click the big fancy button every time you connect you can put the password given into either your server's password field in a profile or use it while connecting.
-
-`To permanently authorise a client, please change your IRC password (server password) to: XXXXXXX`
-
-Remember **NOT TO GIVE YOUR IRC PASSWORD** to anyone else.
 
 Basic IRC Commands
 ==================
```

Comment:
Use heading level 2, there should only be one heading level 1, that being the title.
```
Basic IRC Commands
------------------
```